service/github/github.go

```diff
@@ -123,6 +124,11 @@ func (g *PullRequest) Flush(ctx context.Context) error {
 	return g.postAsReviewComment(ctx)
 }
 
+func (g *PullRequest) SetTool(toolName string, level string) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method PullRequest.SetTool should have comment or be unexported
<!-- __reviewdog__:ChA2YjhjYmZlNzI5ZTg5OTUyEg9nb2xpbnQtY2lyY2xlY2k= -->